packages/dd-trace/test/appsec/iast/analyzers/sql-injection-analyzer.mysql.plugin.spec.js

```diff
@@ -0,0 +1,63 @@
+const { testThatRequestHasVulnerability, testThatRequestHasNotVulnerability } = require('../utils')
+const { storage } = require('../../../../../datadog-core')
+const iastContextFunctions = require('../../../../src/appsec/iast/iast-context')
+const { newTaintedString } = require('../../../../src/appsec/iast/taint-tracking/operations')
+const vulnerabilityReporter = require('../../../../src/appsec/iast/vulnerability-reporter')
+
+describe('sql-injection-analyzer with mysql', () => {
+  let mysql
+  let connection
+  withVersions('mysql', 'mysql', version => {
+    describe('mysql', () => {
+      beforeEach(() => {
+        vulnerabilityReporter.clearCache()
+        mysql = require(`../../../../../../versions/mysql@${version}`).get()
+        connection = mysql.createConnection({
+          host: 'localhost',
+          user: 'root',
+          database: 'db'
+        })
+        connection.connect()
+      })
+
+      afterEach((done) => {
+        connection.end(() => {
+          done()
+        })
+      })
+
+      describe('has vulnerability', () => {
+        testThatRequestHasVulnerability(function () {
```

Comment:
why using anonymous functions all over the place instead of arrow functions ?

Reply:
Any reason really. I'll change it